src/view/home.js

```diff
@@ -177,38 +172,36 @@ const headerStyles = StyleSheet.create({
     width: 150,
     flexDirection: 'row',
     justifyContent: 'flex-start',
-    paddingLeft: 20,
   },
   btnWrapperRight: {
     width: 150,
     flexDirection: 'row',
     justifyContent: 'flex-end',
   },
+  channelBtn: {
+    marginLeft: 3,
+  },
+  settingsBtn: {
+    marginRight: 3,
+  },
   settingsIcon: {
-    height: 17,
-    width: 17,
+    height: 22,
+    width: 22,
   },
 });
 
 const HomeHeader = ({ goChannels, goSettings, showChannelAlert }) => (
   <Header>
-    <View style={headerStyles.btnWrapperLeft}>
-      <SmallButton
-        border
-        alert={showChannelAlert ? color.pinkSig : null}
-        text="Channels"
-        onPress={goChannels}
-      />
-    </View>
+    <Button onPress={goChannels} style={headerStyles.channelBtn}>
+      <ChannelIcon height={24 * 0.9} width={25 * 0.9} />
```

Comment:
I'm getting a different channel icon on desktop vs Android mobile but confused why... Not a problem, just seems like they should be the same.

Reply:
Mmmh. Maybe a caching issue? Try to close the Android emulator and rebuild by restarting the expo job with `npm start`?